code/game/objects/items/stacks/wrap.dm

```diff
@@ -203,3 +210,10 @@
 	w_class = WEIGHT_CLASS_TINY
 	throw_speed = 3
 	throw_range = 5
+
+/obj/item/c_tube/attack(mob/living/target_mob, mob/living/user, params)
+	. = ..()
+	if (istype(target_mob, /mob/living))
```

Comment:
`target_mob` is guaranteed to be living.

Reply:
i don't believe you

Reply:
`attack` only runs for mobs, `attack_atom` only runs for non mobs - that's why the var is cast as living

Reply:
yeah but are you personally guaranteeing nobody is misusing this API ?

Reply:
yes